demo/summer-2016.html

```diff
@@ -234,11 +235,29 @@
         }
       });
     };
+
     editor.setOptions({
         enableBasicAutocompletion: true,
         enableSnippets: true,
-        enableLiveAutocompletion: false
+        enableLiveAutocompletion: true
     });
+   
+    var warmUpCompleter = {
+        getCompletions: function(editor, session, pos, prefix, callback) {
+            var wordList = ["\\begin{itemize} \n \n\\end{itemize}", "\\item"];
+            callback(null, wordList.map(function(word) {
+            return {
+                caption: word,
+                value: word,
+                meta: "TEX"
+            };
+        }));
+        }
+    }
+
+ 
+    editor.completers = [warmUpCompleter];
```

Comment:
So this will completely replace completers and remove one which is built-in, right? Can you combine them somehow? E.g. suggest built-in simple completion if there is no TeX one, or just append simple completion to TeX